examples/CMakeLists.txt

```
@@ -187,3 +187,17 @@ add_test(NAME iotalens.analysis
 )
 
 set_tests_properties(iotalens.analysis PROPERTIES DEPENDS "iotalens.run")
+
+
+# Kurth Beam in a Periodic Channel Test ###################################################
+#
+add_test(NAME kurth_periodic.run
+         COMMAND $<TARGET_FILE:app>${ImpactX_SOURCE_DIR}/examples/kurth/input_kurth_periodic.in
```

Little typo that fails the test:
```suggestion
         COMMAND $<TARGET_FILE:app> ${ImpactX_SOURCE_DIR}/examples/kurth/input_kurth_periodic.in
```